docs/policies/threat_intelligence_policy.md

```diff
@@ -0,0 +1,348 @@
+
+
+### Security monitoring teams
+
+The security monitoring teams are responsible for using threat intelligence to ensure monitoring and detection activity is relevant to the cyber threats that have greatest impact to DfE. For example, through creation of standard detection rules or through focussed threat hunting exercises.
+
+
+### Learning and development and end users
+
+Learning and development teams are responsible for ensuring training and awareness programmes are prioritised based on the threats relevant to the DfE.
+
+End users are responsible for keeping themselves aware of the threats to the department and raising security concerns / reporting security incidents based on these threats.
+
+
+### Supply chain management teams
+
+Supplier and third-party management teams are responsible for ensuring third party vendors and partners:
+
+- share threat intelligence that could impact the DfE’s security posture
+- take appropriate action on threat intelligence passed to them from the DfE
+  
+
+### Responsible Accountable Consulted Informed (RACI) matrix
+
+| Role<br><br>Activity | CISO | SRO | CTI team | Project/<br><br>Operations security leads | Development/Operations teams | Incident Management teams | Security monitoring<br><br>teams | End Users | Supply Chain teams | Learning & Development |
+| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
+| Define and enable strategy | A   | C   | I   | I   | I   | I   | I   | I   | I   | I   |
+| Ensure teams make use of threat intelligence | \-  | A   | I   | R   | R   | R   | R   | \-  | R   | R   |
+| Collect, collate, analyse and distribute threat intelligence | C   | C   | R   | R   | C   | C   | C   | \-  | C   | \-  |
+| Coordinate threat intelligence activity | \- | A   | R   | R   | C   | C   | C   | \-  | R   | \-  |
+| Be aware of threat intelligence | \- | A   | R   | I   | I   | I   | I   | I   | I   | I   |
+| Utilise threat intelligence to detect vulnerabilities | A   | A   | C   | C   | R   | R   | R   | \-  | R   | \-  |
+| Utilise threat intelligence to design out vulnerabilities | A   | A   | C   | C   | R   | R   | R   | \-  | R   | \-  |
+| Utilise threat intelligence to manage incidents | A   | A   | C   | R   | R   | R   | C   | \-  | R   | \-  |
+| Feedback usefulness of threat intelligence | \- | A   | R   | R   | C   | C   | C   | C   | C   | C   |
+| Report on threat intelligence | I   | I   | A   | R   | C   | C   | C   | \-  | C   | C   |
+| Arrange TI sharing agreements with OGDs and ALBs | A   | C   | R   | \-  | \-  | \-  | I   | \-  | C   | R   |
+| Arrange training based on TI | A   | R   | C   | I   | I   | I   | I   | I   | I   | \- |
+
+
+## Exceptions
+Exceptions to this policy are likely to occur. Exception requests shall be made by email to the policy owner identified in the [Revision history](#revision_history) and must contain:
+
+- the reason for the request
+- risk to the DfE of not following the written policy
```

Comment:
Same as above re removing 'the' in front of DfE. I'm sure I removed 'written' from in front of 'policy' for previous versions/other policies, but not the end of the world if it stays - I just think it's obvious it's written!